# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 04d6fcc72269
# Full Hash: 04d6fcc72269f62e73cecedca1bde49d1aebbfda
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2022-12-13 04:11:09
# Regressor Bug: 1451394
# File Overlap Count: 1
# Description:
#   Bug 1451394 - Enable new Mac camera backend in Nightly and early Beta. r=webrtc-reviewers,jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D163686
# ==============================================================================

diff -r 703360f0b861 -r 04d6fcc72269 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Mon Dec 12 15:47:22 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Mon Dec 12 15:47:22 2022 +0000
@@ -10359,7 +10359,7 @@
 # disabled, an older forked capture module is used.
 - name: media.getusermedia.camera.macavf.enabled
   type: bool
-  value: false
+  value: @IS_EARLY_BETA_OR_EARLIER@
   mirror: once
 
 # WebRTC prefs follow
